vispy/visuals/isoline.py

```diff
@@ -215,4 +208,4 @@ def draw(self, transforms):
             self._update_color_lev = False
 
         LineVisual.set_data(self, pos=self._v, connect=self._c, color=self._cl)
```

Comment:
This looks expensive--ideally, `set_data` is only called if something has changed. I'd be happy to merge your changes as they are (especially since this was the state of the code before you got to it), but if you happen to have time to take care of this line, that'd be awesome.